### ISSUE TITLE
v11 not starting on macOS

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#3071]
From @leff32
In my case UMS won't start after updating to 11.0.0 or 11.1.0.
Had to roll back to 10.21.1 which works fine.

I suspect it may be because of the JRE you mentioned here.

I'm running macOS 12.4.
Debug.log shows just 6 lines before app quits.
```
DEBUG 2022-06-20 10:28:02.225 [main] LogBack started with configuration file: /Applications/Universal Media Server.app/Contents/Resources/logback.xml
DEBUG 2022-06-20 10:28:02.228 [main] Starting services
DEBUG 2022-06-20 10:28:02.228 [main] Starting ProcessManager
DEBUG 2022-06-20 10:28:02.230 [Process Terminator] ProcessTerminator is starting
DEBUG 2022-06-20 10:28:02.231 [main] Reading prevent_sleep: "RUNNING" (default: "PLAYBACK")
DEBUG 2022-06-20 10:28:02.231 [main] Starting SleepManager
```

Any ideas?

**Comment by Demonsthere** [comment]
Running it from the binary:

```shell
 ./UMS                                                                                                             15:09   03.07.22 
15:09:41,529 |-INFO in ch.qos.logback.classic.joran.action.ConfigurationAction - debug attribute not set
15:09:41,530 |-INFO in ch.qos.logback.core.joran.action.DefinePropertyAction - About to instantiate property definer of type [net.pms.logging.DebugLogPropertyDefiner]
15:09:41,536 |-INFO in ch.qos.logback.core.joran.action.DefinePropertyAction - Popping property definer for property named [logFilePath] from the object stack
15:09:41,537 |-WARN in Logger[net.pms.configuration.PmsConfiguration] - No appenders present in context [default] for logger [net.pms.configuration.PmsConfiguration].
15:09:41,537 |-INFO in ch.qos.logback.core.joran.action.DefinePropertyAction - About to instantiate property definer of type [net.pms.logging.DebugLogPropertyDefiner]
15:09:41,537 |-INFO in ch.qos.logback.core.joran.action.DefinePropertyAction - Popping property definer for property named [rootLevel] from the object stack
15:09:41,538 |-INFO in ch.qos.logback.core.joran.action.DefinePropertyAction - About to instantiate property definer of type [net.pms.logging.DebugLogPropertyDefiner]
15:09:41,538 |-INFO in ch.qos.logback.core.joran.action.DefinePropertyAction - Popping property definer for property named [logFileName] from the object stack
15:09:41,538 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - About to instantiate appender of type [net.pms.logging.FrameAppender]
15:09:41,538 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - Naming appender as [traces]
15:09:41,551 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - About to instantiate appender of type [ch.qos.logback.core.ConsoleAppender]
15:09:41,551 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - Naming appender as [console]
15:09:41,552 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - About to instantiate appender of type [net.pms.logging.UmsFileAppender]
15:09:41,554 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - Naming appender as [default.log]
15:09:41,559 |-INFO in net.pms.logging.UmsFileAppender[default.log] - The previous log will be compressed to [/Users/jakubblaszczyk/Library/Application Support/UMS/debug.log.prev.zip].
15:09:41,559 |-INFO in net.pms.logging.UmsCompressor - ZIP compressing [/Users/jakubblaszczyk/Library/Application Support/UMS/debug.log] as [/Users/jakubblaszczyk/Library/Application Support/UMS/debug.log.zip]
15:09:41,569 |-INFO in net.pms.logging.UmsCompressor - Compressing [/Users/jakubblaszczyk/Library/Application Support/UMS/debug.log] take PT0.010083S
15:09:41,572 |-WARN in net.pms.logging.UmsFileAppender[default.log] - Append mode is mandatory for RollingFileAppender. Defaulting to append=true.
15:09:41,572 |-INFO in net.pms.logging.UmsFileAppender[default.log] - Active log file name: /Users/jakubblaszczyk/Library/Application Support/UMS/debug.log
15:09:41,572 |-INFO in net.pms.logging.UmsFileAppender[default.log] - File property is set to [/Users/jakubblaszczyk/Library/Application Support/UMS/debug.log]
15:09:41,572 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [org.apache] to WARN
15:09:41,572 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [httpclient.wire] to WARN
15:09:41,572 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [fm.last.musicbrainz.coverart] to WARN
15:09:41,572 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [oshi] to WARN
15:09:41,572 |-INFO in ch.qos.logback.classic.joran.action.RootLoggerAction - Setting level of ROOT logger to ALL
15:09:41,572 |-INFO in ch.qos.logback.core.joran.action.AppenderRefAction - Attaching appender named [traces] to Logger[ROOT]
15:09:41,572 |-INFO in ch.qos.logback.core.joran.action.AppenderRefAction - Attaching appender named [console] to Logger[ROOT]
15:09:41,572 |-INFO in ch.qos.logback.core.joran.action.AppenderRefAction - Attaching appender named [default.log] to Logger[ROOT]
15:09:41,572 |-INFO in ch.qos.logback.classic.joran.action.ConfigurationAction - End of configuration.
15:09:41,572 |-INFO in ch.qos.logback.classic.joran.JoranConfigurator@4f18837a - Registering current configuration as safe fallback point

Exception in thread "main" java.lang.NullPointerException: Cannot invoke "net.pms.service.SleepManager$AbstractSleepWorker.preventSleep()" because "this.worker" is null
	at net.pms.service.SleepManager.preventSleep(SleepManager.java:236)
	at net.pms.service.SleepManager.start(SleepManager.java:185)
	at net.pms.service.SleepManager.<init>(SleepManager.java:84)
	at net.pms.service.Services.start(Services.java:156)
	at net.pms.service.Services.<init>(Services.java:140)
	at net.pms.service.Services.create(Services.java:96)
	at net.pms.PMS.main(PMS.java:1071)
```

**Comment by SubJunk** [comment]
That's a useful log, we should be able to fix it now. I will make a test file for you soon @Demonsthere

**Comment by SubJunk** [comment]
@Demonsthere please try https://www.universalmediaserver.com/uploads/UMS-3071.dmg

**Comment by Demonsthere** [comment]
Hui @SubJunk, the hotfix works and I can run the app :) However, i do run into a migration error, judging from the logs during data migration from the previous version:

```
10:14:48 ERROR Database "medias" error while writing in "VIDEO_METADATA_POSTERS" for "/Users/foobar/Movies/The Spine Of Night (2021) [1080p] [BluRay] [5.1] [YTS.MX]/The.Spine.Of.Night.2021.1080p.BluRay.x264.AAC5.1-[YTS.MX].mp4": Unique index or primary key violation: "PUBLIC.FILENAME_POSTER_TVSERIESID_IDX ON PUBLIC.VIDEO_METADATA_POSTERS(FILENAME NULLS FIRST, TVSERIESID NULLS FIRST) VALUES ( /* key:34 */ '/Users/foobar/Movies/The Spine Of Night (2021) [1080p] [BluRay] [5.1] [YTS.MX]/The.Spine.Of.Night.2021.1080p.BluRay.x264.AAC5.1-[YTS.MX].mp4', -1)"; SQL statement:
INSERT INTO VIDEO_METADATA_POSTERS (TVSERIESID, FILENAME, POSTER) VALUES (?, ?, ?) [23505-214]
```

**Comment by SubJunk** [comment]
Thanks for confirming the fix!

The first new problem you mentioned is something we are aware of, it doesn't seem to cause any bugs other than an ugly log entry (the poster is still set one time, but it fails the second time, I think. There is an issue for it at https://github.com/UniversalMediaServer/UniversalMediaServer/issues/3132

The second new problem seems like it is common in Java 17 (which we started using in UMS v11) but again it doesn't seem to cause any actual bugs for users. I saw some [mention](https://github.com/processing/processing4/issues/368) that it just goes away from logs in Java 18
